api/lagoon/client/_lgraphql/deploytargets/deployTargetsByOrganizationId.graphql

```diff
@@ -0,0 +1,21 @@
+query (
+  $id: Int!) {
+    organizationById(
+    id: $id) {
+    deployTargets {
+      id
+      name
+      created
+      consoleUrl
+      cloudRegion
+      cloudProvider
+      friendlyName
+      token
```

Comment:
remove the `token`, `consoleUrl`, and `monitoringConfig` from this query, as these are admin only

Reply:
Do this for all instances where these may be used